app/services/subscriptions/update_service.rb

```diff
@@ -61,6 +60,22 @@ def process_subscription_at_change(subscription)
       BillSubscriptionJob.perform_later([subscription], Time.current.to_i)
     end
 
+    def handle_plan_override
+      current_plan = subscription.plan
+
+      if current_plan.parent_id
+        Plans::UpdateService.call(
+          plan: current_plan,
+          params: params[:plan_overrides].to_h.with_indifferent_access,
+        )
+      else
+        Plans::OverrideService.call(
+          plan: current_plan,
+          params: params[:plan_overrides].to_h.with_indifferent_access
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)